app/views/organization_wizzard/new_account.html.slim

```diff
@@ -0,0 +1,29 @@
+.page-header
+  h2 New organization bank account
+
+.row.previous_buttons class="#{flash[:open_form] ? 'hidden' : ''}"
+  .col-sm-6
+    p
+      ' Bank account will be created by default (with default currency)
+      br
+      br
+      = button_to 'Create default Bank account', default_account_organization_path(@organization.id), class: 'btn btn-primary', method: :get
+  .col-sm-6
+    p
+      ' But you can create it manually
```

Comment:
@dissident User should read "[Create default Bank account] or [Create it manually]". [] <- buttons